app/src/main/java/be/scri/services/SimpleKeyboardIME.kt

```diff
@@ -65,13 +65,8 @@ abstract class SimpleKeyboardIME :
 
     override fun hasTextBeforeCursor(): Boolean {
         val inputConnection = currentInputConnection ?: return false
-        val textBeforeCursor = inputConnection.getTextBeforeCursor(Int.MAX_VALUE, 0)
-        if (textBeforeCursor.isNullOrBlank()) {
-            return false
-        }
-        val trimmedText = textBeforeCursor.trim()
-        val lastChar = trimmedText.lastOrNull()
-        return lastChar != '.'
+        val textBeforeCursor = inputConnection.getTextBeforeCursor(Int.MAX_VALUE, 0).orEmpty().trim()
```

Comment:
Instead of using `isEmpty()`, I recommend performing a simple null check in the return statement. This adjustment should help avoid any errors.

Additionally, I suggest running the application in Android Studio  or following the steps outlined in the new PR template, specifically `./gradlew lintKotlin detekt test`. This could help catch potential issues early on. 

Thank you!